src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -6,7 +6,7 @@
     </h1>
   </a>
   <div class="searchbox">
-    <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
+    <div class="gcse-searchbox-only" enableAutoComplete="true" data-resultsUrl="/search"></div>
```

Comment:
Let's be really careful that this PR does not automatically close the searchbox suggestions issue. I'm not sure if it will or not, but the phrase "fix 75" (with the hashtag) does appear in the PR (albeit prefixed with "did not"), so it might get closed anyhow.